docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.5'
+
+services:
+
+  qmk_api:
+    build: .
+    ports:
+     - "5000:5000"
+    environment:
+      S3_ACCESS_KEY: minio_dev
+      S3_SECRET_KEY: minio_dev_secret
+      REDIS_HOST: redis
+      S3_HOST: http://minio:9000
```

Comment:
Alphabetically doesn't make as much sense in this instance. That would order it as "access key", "hostname", "secret key", and splitting up the username/password combination that way is not intuitive. 

If we're nitpicking I would swap the order of `REDIS_HOST` and `S3_HOST`, but otherwise these are in a logical order that is better. With only 5 items the utility of alphabetical ordering is very minimal.

Reply:
I would actually move the `REDIS_HOST` to the top and have the S3 stuff below them, but I agree, pure alphabetical order is probably worse in this instance.

Reply:
Wasn't strictly shooting for alphabetical ordering — anything works as long as `REDIS_HOST` isn't in the middle of  `S3_*` vars.

Reply:
By the way, (hostname, access key, secret key) makes the most sense imo.

Reply:
Yeah, that's a reasonable order imo. I'll update to that.